app/src/main/res/layout/activity_main.xml

```diff
@@ -16,129 +16,93 @@
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toTopOf="@id/ll_paint_colors">
+        app:layout_constraintBottom_toTopOf="@id/colorPaletteLayout">
 
         <ImageView
-            android:id="@+id/iv_background"
+            android:id="@+id/imageViewBackground"
             android:scaleType="centerCrop"
             android:contentDescription="background_image"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:src="@drawable/image"/>
 
         <media.taylor.drawingapp.DrawingView
+            android:id="@+id/drawingView"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:id="@+id/drawing_view"
             android:background="#80FFFFFF" />
 
     </FrameLayout>
 
 
     <LinearLayout
-        android:id="@+id/ll_paint_colors"
+        android:id="@+id/colorPaletteLayout"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:orientation="horizontal"
-        app:layout_constraintBottom_toTopOf="@id/ll_action_buttons"
+        app:layout_constraintBottom_toTopOf="@id/actionButtonLayout"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/fl_drawing_view_container">
+        app:layout_constraintTop_toBottomOf="@id/drawingViewLayout">
 
         <ImageButton
             android:tag="@color/skin"
-            android:layout_width="25dp"
-            android:layout_height="25dp"
-            android:layout_margin="2dp"
-            android:onClick="paintClicked"
-            android:contentDescription="image"
             android:background="@color/skin"
-            android:src="@drawable/palette_normal"/>
+            android:contentDescription="skin color palette"
```

Comment:
calling this shade as "skin color" perpetuates that whiteness is the "regular" skin tone !